### PR TITLE
[docs] PR for Move CDC from Explore to Integrations (Doc9188)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Moved "CDC" under Integrations, including "CDC for Kafka" from Deploy.
Attn: "Change Data Capture to Kafka" shows under Integrations in the left-side menu. This is an error because both cdc-generic.md and cdc-kafka.md need to be under "Change Data Capture" and then toggled via switch buttons.

### CODE REVIEW

File: docs/content/latest/integrations/change-data-capture/cdc-generic.md [hunk 1]
```diff
@@ -176,7 +173,7 @@ changes {
 
 ## 6. Clean up (optional)
 
-Optionally, you can shut down the local cluster you created.
+Optionally, you can shut down the local cluster by executing the followingn command:
```
Comment (ddhodge) [inline docs/content/latest/integrations/change-data-capture/cdc-generic.md:177]: following

File: docs/content/latest/integrations/change-data-capture/cdc-generic.md [hunk 2]
```diff
@@ -1,61 +1,60 @@
...
-      Linux
+    <a href="{{< relref "./cdc-kafka.md" >}}" class="nav-link">
+      Kafka
     </a>
   </li>
 
 </ul>
 
-[Change data capture (CDC)](../../../architecture/cdc-architecture) can be used to asynchronously stream data changes from a YugabyteDB cluster to external systems like message queues and OLAP warehouses. The data changes in YugabyteDB are detected, captured, and then output to the specified target.  In the steps below, you will use a local YugabyteDB cluster to stream data changes to `stdout` using the CDC API.
+[Change data capture (CDC)](../../../architecture/cdc-architecture) can be used to asynchronously stream data changes from a YugabyteDB cluster to external systems such as message queues and OLAP warehouses. The data changes in YugabyteDB are detected, captured, and then directed to a specified target.
```
Comment (ddhodge) [inline docs/content/latest/integrations/change-data-capture/cdc-generic.md:36]: link is now actually architecture/docdb-replication/change-data-capture/ kept alive by an alias I guess

File: docs/content/latest/integrations/change-data-capture/cdc-kafka.md [hunk 3]
```diff
@@ -1,30 +1,49 @@
...
+</ul>
+
+You can enable a YugabyteDB cluster to use the change data capture [(CDC)](../../../architecture/cdc-architecture) API to send data changes to Apache Kafka.
 
 ## Prerequisites
 
 ### YugabyteDB
 
-Create a YugabyteDB cluster using the steps outlined in [Manual Deployment](../../manual-deployment/).
+You can create a YugabyteDB cluster using the steps outlined in [Manual Deployment](../../manual-deployment/).
```
Comment (ddhodge) [inline docs/content/latest/integrations/change-data-capture/cdc-kafka.md:43]: link is broken

**Review by ddhodge — APPROVED** [review]
minor fixes

File: docs/content/latest/integrations/change-data-capture/cdc-generic.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/change-data-capture/cdc-generic.md:177]: fixed

File: docs/content/latest/integrations/change-data-capture/cdc-generic.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/change-data-capture/cdc-generic.md:36]: changed

File: docs/content/latest/integrations/change-data-capture/cdc-kafka.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/change-data-capture/cdc-kafka.md:43]: fixed

**Pull request merged**